public-ui/src/app/components/db-no-results/no-results-search.component.tsx

```diff
@@ -0,0 +1,184 @@
+import { Component, Input } from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { LoadingDots } from 'app/utils/spinner';
+import { environment } from 'environments/environment';
+import { domainToRoute, surveyIdToRoute } from 'app/utils/constants';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+import {navigate, navigateByUrl} from 'app/utils/navigation';
+import { reactStyles } from 'app/utils/index';
+
+const api = new DataBrowserApi(new Configuration({ basePath: environment.publicApiUrl }));
+
+const styles = reactStyles({
+    loadingText: {
+        fontFamily: 'GothamBook, Arial, sans-serif',
+        fontWeight: 'normal',
+        fontStyle: 'normal',
+        fontStretch: 'normal',
+        lineHeight: '1.5',
+        fontSize: '16px',
+        letterSpacing: 'normal',
+        textAlign: 'left',
+        color: '#262262',
+        paddingLeft: '0'
+  },
+  noResults: {
+    marginTop: '-1rem',
+    padding: '1em'
+  },
+  loadingDiv: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'flex-start',
+      alignItems: 'center',
+      marginBottom: '1em'
+  },
+  spinnerDiv: {
+      marginTop: '1em',
+      marginBottom: '0.2em',
+      marginLeft: '0.5em'
+  }
```

Comment:
Nit: consistent indentation
```suggestion
    },
    noResults: {
        marginTop: '-1rem',
        padding: '1em'
    },
    loadingDiv: {
        display: 'flex',
        flexDirection: 'row',
        justifyContent: 'flex-start',
        alignItems: 'center',
        marginBottom: '1em'
    },
    spinnerDiv: {
        marginTop: '1em',
        marginBottom: '0.2em',
        marginLeft: '0.5em'
    }
```

Reply:
Thanks changed this.